ios/PurchasesHybridCommon/Podfile.lock

```diff
@@ -1,14 +1,14 @@
 PODS:
   - Nimble (10.0.0)
   - Quick (5.0.1)
-  - RevenueCat (4.7.0)
+  - RevenueCat (4.8.0)
   - SnapshotTesting (1.9.0)
 
 DEPENDENCIES:
   - Nimble
   - Quick
-  - RevenueCat (= 4.7.0)
-  - "SnapshotTesting (from `git@github.com:pointfreeco/swift-snapshot-testing.git`, branch `main`)"
+  - RevenueCat (= 4.8.0)
+  - SnapshotTesting (from `https://github.com/pointfreeco/swift-snapshot-testing.git`, branch `main`)
```

Comment:
This might lead to issues in CI. Did you mean to change it?

Reply:
oops, the git thing was failing on my machine so I updated to https locally but missed that it was there in the podfile.lock too. I'll revert this bit